Login.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+session_start();
+
+require_once 'src/models/UsersModel.php';
+require_once 'src/services/DatabaseConnectionService.php';
+
+$db = DatabaseConnectionService::connect();
+
+if (!isset($_SESSION['loggedIn'])) {
+    if (isset($_POST['submit'])) {
+        $username = $_POST['username'];
+        $password = $_POST['password'];
+        $UsersModel = new UsersModel($db);
+        $login = $UsersModel->login($username, $password);
+        if ($login) {
+            $_SESSION['loggedIn'] = true;
+            $_SESSION['username'] = $username;
+            header('Location: index.php');
```

Comment:
You should call exist(); after redirecting with header.

It's just an extra check to make sure the rest of the code doesn't run if the users browser doesn't redirect them (really this only applies to search engine bots, but it's still best practice)

Reply:
* exit(), not exist()